pkgs/os-specific/bsd/freebsd/lib/default.nix

```diff
@@ -15,4 +19,61 @@
     .${stdenv'.hostPlatform.parsed.cpu.name} or stdenv'.hostPlatform.parsed.cpu.name;
 
   install-wrapper = builtins.readFile ../../lib/install-wrapper.sh;
+
+  filterPatches =
```

Comment:
This is too long. Can it be at least documented?

Reply:
It does need documentation, but do note it is just being moved. I would be OK with that happening in a later PR.

Reply:
I added an overview.